pcsx2-qt/GameList/GameListWidget.cpp

```diff
@@ -435,9 +441,8 @@ void GameListWidget::processBackgroundFrames()
 
 		QtUtils::resizeAndScalePixmap(&pm, widget_width, widget_height, dpr, m_background_scaling, m_background_opacity);
 
-		QPalette bg_palette(m_ui.stack->palette());
-		bg_palette.setBrush(QPalette::Base, pm);
-		m_ui.stack->setPalette(bg_palette);
+		m_background_pixmap = pm;
```

Comment:
You should use std::move here to avoid a deep copy.

Reply:
alr fixed now